sdcm/db_stats.py

```diff
@@ -619,7 +619,7 @@ def get_test_details(self):
         test_details['job_url'] = get_job_url()
         test_details['start_host'] = platform.node()
         test_details['test_duration'] = self.params.get(key='test_duration')
-        test_details['start_time'] = time.time()
+        test_details['start_time'] = int(time.time())
```

Comment:
if this is going to be use as the timestamp of the data, why not use `datetime.utcnow()` ?

Reply:
in many places where calculating timeintervals and in regression stats calculate used time.time() for end timestamp. To avoid time conversion in different places, i set such way
It could be done in another pr
or maybe next constructiion could be used:
`int(time.mktime(datetime.datetime.utcnow().utctimetuple()))`

Reply:
O.k.